admin/scripts/zvlastni/prezence-tisk.xtpl

```diff
@@ -41,7 +41,7 @@
   <table>
     <tr>
       <th colspan="3" class="hlavicka">
-        <div class="nazev">{a.nazev}</div>
+        <div class="nazev">{a.typ} - {a.nazev}</div>
```

Comment:
nevytiskne to číslo místo názvu typu?

Reply:
Vytiskne, měl jsem to v otaznících v trello kartě

Reply:
zatím vyhodit typ a přidat příp. v dalším PR pokud tam dodáme trait a vypíšem přímo název linie